### PR TITLE
build: allow DRIVER_REVISION to be of any length, but truncate it to 12

### PR DESCRIPTION
**Pull request by jazzdan**
Related BAC-4852

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -48,10 +48,11 @@ if (!REPLAY_LOCAL_DRIVER_DIR) {
   let driverArchive = `${currentPlatform()}-recordreplay${archSuffix}.tgz`;
   let downloadArchive = driverArchive;
   let driverRevisionOverride = DRIVER_REVISION;
+  driverRevisionOverride = driverRevisionOverride.substring(0, 12);
```
Comment (toshok) [inline build.js:51]: can we remove the duplicate line down on line 58 now?

**Review by toshok — APPROVED** [review]

File: build.js (hunk 1 above)
Comment (jazzdan) [reply build.js:51]: whoops, yes. Good call!

File: build.js [hunk 2]
```diff
@@ -48,13 +48,13 @@ if (!REPLAY_LOCAL_DRIVER_DIR) {
...
   if (driverRevisionIsSet) {
-    if (driverRevisionOverride.length < 12) {
+    if (driverRevisionOverride.length === 12) {
```
Comment (Domiii) [inline build.js:53]: Hum... So if its 12, it'll throw?

File: build.js (hunk 1 above)
Comment (Domiii) [reply build.js:51]: Is this guaranteed to never be `undefined`?

**Pull request merged**